apps/demo-app/i18n/locales/de-DE.json

```diff
@@ -13,7 +13,42 @@
   "action": "Aktion | Aktionen",
   "dataGrid": "Data Grid / Tabelle",
   "forms": "Formulare",
-  "charts": "Diagramme",
+  "charts": {
```

Comment:
The charts translation key is missing on the home page
<img width="353" height="206" alt="image" src="https://github.com/user-attachments/assets/66d14bc1-1696-4d71-99b1-b7efbc636a4a" />